Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -140,13 +111,18 @@ void BinaryBHLevel::specificPostTimeStep()
     CH_TIME("BinaryBHLevel::specificPostTimeStep");
     if (m_p.activate_extraction == 1)
     {
+        int min_level = m_p.extraction_params.min_extraction_level();
+        bool calculate_weyl = at_level_timestep_multiple(min_level);
+        if (!calculate_weyl)
+            return;
```

Comment:
Won't this stop puncture tracking? I don't think you should `return` this function but rather just guard the relevant code in a block conditional on this.